test/integration_tests/test_select_executor.py

```diff
@@ -247,6 +247,21 @@ def test_select_and_limit(self):
         self.assertEqual(len(actual_batch), len(expected_batch[0]))
         self.assertEqual(actual_batch, expected_batch[0])
 
+    def test_select_and_aggregate(self):
+        simple_aggregate_query = "SELECT COUNT(*), AVG(id) FROM MyVideo;"
+        actual_batch = execute_query_fetch_all(simple_aggregate_query)
+
+        self.assertEqual(actual_batch.frames.iat[0, 0], 10)
+        self.assertEqual(actual_batch.frames.iat[0, 1], 4.5)
+
+        complex_aggregate_query = """SELECT SUM(id), COUNT(label)
```

Comment:
We should add a test case with aggregate on ndarray column.

Reply:
A weird part is that when the query operates on an ndarray column, it does not reduce it to a single row. It actually keeps as many rows around as the number of the rows in the input column. Does that even make sense?

Reply:
Should we support the `NDARRAY` case?

Reply:
I have reverted back the `NDARRAY` case as it does not make sense.

Reply:
Do we raise an error if the query tries to aggregate on an array column? We should add a test case to verify it. Thanks!